### ISSUE TITLE
Non-existing env vars break the commands in Windows

### ISSUE THREAD

**Issue by DanReyLop** [kentcdodds/cross-env#145]
- `cross-env` version: `5.0.5`
- `node` version: `6.11.1`
- `npm` (or `yarn`) version: `npm@3`

We have this code: https://github.com/Automattic/wp-calypso/blob/master/package.json#L224
To simplify:
```
{
  "scripts": {
    "start": "cross-env-shell node $NODE_ARGS index.js"
  }
}
```

The idea is that the app can be started like this, if needed:
`NODE_ARGS="--debug-brk"; npm start`
Or any other `node` flag.

- In Linux, the command converted to: `node $NODE_ARGS index.js`
- In Windows: `node %NODE_ARGS% index.js`

When running `NODE_ARGS="--debug-brk"; npm start`,
- In Linux: `node --debug-brk index.js`
- In Windows: `node --debug-brk index.js` (correct)

When running plain `npm start`,
- In Linux: `node  index.js`
- In Windows: `node %NODE_ARGS% index.js` 💥 

To sum up, in UNIX, a non-existing environment variable resolves to an empty string, while in Windows, a non-existent environment variable isn't replaced by the shell.

The solution would be to parse the command and replace the non-existing env variables with an empty string (the UNIX behaviour).

The line between "missing feature" and "bug" is blurry here. I'd argue that this is a bug. What do you think? I'll be happy to send a PR if it's not gonna be rejected because this project is in "maintenance only" :)

**Comment by kentcdodds** [comment]
Thanks Daniel! I agree that this is a bug and I'd love a pull request for it!
